src/EFCore.MySql/Migrations/MySqlMigrationsSqlGenerator.cs

```diff
@@ -279,8 +279,7 @@ protected override void Generate(AlterTableOperation operation, IModel model, Mi
             if (!_options.ServerVersion.Supports.Sequences)
             {
                 throw new InvalidOperationException(
-                    $"Error in Restart Sequence {operation.Name}: Sequences are not supported." /* +
-                    // $"Sequences are  supported only from  {ServerVersion.Supports.GetSupport(ServerVersion.SequencesSupportKey)}." */); 
+                    $"Cannot create sequence '{operation.Name}' because sequences are not supported in server version {_options.ServerVersion}.");
```

Comment:
This should be `Cannot restart sequence ...`.

Reply:
Done, sorry for my sloppiness and thank you for your patience

Reply:
No worries, all good!